stamp_service/fits2png.py

```diff
@@ -6,8 +6,12 @@
 
 
 def _read_compressed_fits(compressed_fits_file):
-    fits = io.BytesIO(compressed_fits_file)
-    return fio.open(gzip.open(fits))[0]
+    try:
+        fits = io.BytesIO(compressed_fits_file)
+        return fio.open(gzip.open(fits))[0]
+    except Exception as e:
```

Comment:
Why not go only for the specific expected error? I think it is OSError

Reply:
will do. and yes its os error